VTUL/dlp-access
# LIBTD-1968: Provide more viewing options in search results: Gallery a…

…nd Masonry

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1968 and https://webapps.es.vt.edu/jira/browse/LIBTD-1969) (:star:)

# What does this Pull Request do? (:star:)
This PR provides three viewing options (list, gallery and masonry) for displaying search results and items list.

# What's the changes? (:star:)

* Added three viewing pages for list, gallery, and masonry, respectively
* Added state to hold chosen viewing option

# How should this be tested?

* Go to "Browse items" and toggle between three viewing buttons with tooltips 
* Enter partial title text in the search box, then click "GO" button to see search results with previous chosen viewing option. Also toggle between different viewing buttons to observe results being displayed differently. 

# Interested parties
Tag (@yinlinchen ) interested parties

(:star:) Required fields
